CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Braintree Android SDK Release Notes
 
+## unreleased
+
+* Breaking Changes
```

Comment:
I don't think this should be a breaking change

Reply:
Thanks for the heads up. I just moved the entire block, but I should have confirmed first. I’ll keep that in mind for the future. Updated: [0f21f4d](https://github.com/braintree/braintree_android/pull/1174/commits/0f21f4df557fb4556a4b6a756e76508bd3bbb071)